### PR TITLE
Prep for release `v2.0.0`

### LINKED ISSUE #578: Add a comment module ui to `Source Explorer`'s File Browser

### PR DESCRIPTION
**Pull request by AARON-CLARK**
Waiting on #480 before merge.

This PR goes through all of the points of: https://github.com/pharmaR/riskassessment/wiki/Cutting-a-new-release

This PR also updates the 'get started' vignette and closes #578

### CODE REVIEW

File: R/mod_pkg_explorer.R
Comment (Jeff-Thompson12) [inline R/mod_pkg_explorer.R:1]: @AARON-CLARK you borked the jsTreeR by using the `uiOutput()`

Comment (Jeff-Thompson12) [reply R/mod_pkg_explorer.R:1]: Here's a potential patch for the issue. Even more reason for us to move away from `{jsTreeR}` as soon as possible.
[patch.txt](https://github.com/pharmaR/riskassessment/files/12253360/patch.txt)

Comment (AARON-CLARK) [reply R/mod_pkg_explorer.R:1]: @Jeff-Thompson12 unborked with commit: https://github.com/pharmaR/riskassessment/pull/606/commits/53afd55672946bc7d893d94a1fa285058cbcd7bd

**Review by Jeff-Thompson12 — APPROVED** [review]
LGTM!

**Pull request merged**